src/main/resources/reference.conf

```diff
@@ -616,7 +622,7 @@ resourceTypes = {
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["update_billing_account", "update_metadata", "delete", "link", "share_policy::owner", "share_policy::user"]
+        roleActions = ["update_billing_account", "update_metadata", "delete", "link", "share_policy::owner", "share_policy::user", "read_policies", "alter_policies"]
```

Comment:
Ok, thank you! Do you think we cover all of the actions that alter_policies would cover? It doesn't look like we have an action to change roles. (But, I'm not sure if that's needed)

Reply:
You guys are the ones creating the policies when you create the resource. Once created you are not giving anyone permissions to change what policies exist or the roles they have. This is the current best practice because it leaves your app in control of the structure but still allows users to change memberships.